src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/SdkTemplateTests.cs

```diff
@@ -35,7 +35,7 @@ public void VerifyConsoleTemplate(DotNetLanguage language)
     [Theory]
     [MemberData(nameof(GetLanguages))]
     [Trait("Category", "Offline")]
-    public void VerifyOfflineConsoleTemplate(DotNetLanguage language)
+    public void VerifyConsoleTemplate(DotNetLanguage language)
     {
         var newTest = new SdkTemplateTest(
             nameof(SdkTemplateTests) + "Offline", language, _scenarioTestInput.TargetRid, DotNetSdkTemplate.Console,
```

Comment:
`"nameof(SdkTemplateTests) + "Offline"` should be refactored accordingly as well.

Reply:
Ack. Yes it should be.

Reply:
Fixed in latest commit